backend/src/main/java/edu/nus/market/controller/AccountController.java

```diff
@@ -59,41 +55,52 @@ public ResponseEntity<Object> register(@Valid @RequestBody Register register, Bi
     }
 
     @DeleteMapping("/me")
-    public ResponseEntity<Object> deleteAccount(@Valid @RequestBody DelAccReq req, BindingResult bindingResult, @RequestHeader("Set-Cookie") String token){
+    public ResponseEntity<Object> deleteAccount(@Valid @RequestBody DelAccReq req, BindingResult bindingResult, @RequestHeader("Cookie") String token){
         if (bindingResult.hasErrors()) {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new ErrorMsg(ErrorMsgEnum.INVALID_DATA_FORMAT.ErrorMsg));
         }
         //System.out.println(JwtTokenManager.decodeAccessToken(token));
         if (!JwtTokenManager.validateCookie(token)) {
             return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(new ErrorMsg(ErrorMsgEnum.UNAUTHORIZED_ACCESS.ErrorMsg));
         }
-        return accountService.deleteAccountService(req);
+        return accountService.deleteAccountService(req, JwtTokenManager.decodeCookie(token));
     }
 
     @PatchMapping("/me/profile")
```

Comment:
It's `PATCH /auth/me`, not `PATCH /auth/me/profile`.
```suggestion
    @PatchMapping("/me")
```